src/bicome/controllers/ReflectionController.java

```diff
@@ -20,7 +20,7 @@
 import javafx.event.ActionEvent;
 
 
-public class ReflectionController { //implements Initializable {
+public class ReflectionController implements Initializable {
```

Comment:
Where is
`initialize(URL location, ResourceBundle resources)`
 method?
It has to be implemented in order to make class Initializable